client/extensions/woocommerce/app/products/product-create.js

```diff
@@ -88,14 +89,19 @@ class ProductCreate extends React.Component {
 	}
 
 	render() {
-		const { siteId, product, className, variations, productCategories } = this.props;
+		const { siteId, product, className, variations, productCategories, actionList } = this.props;
+
+		const isReady = 'undefined' !== siteId;
```

Comment:
This is the same issue as @justinshreve mentions in the previous PR: https://github.com/Automattic/wp-calypso/pull/14559#pullrequestreview-44900257

While it's not a direct artifact of this PR or the last, I took care of it in the [latest commit](https://github.com/Automattic/wp-calypso/pull/15174/commits/74a0c4fe8f53d4d82810b6a61d332b530febcb64)